### DISCUSSION TITLE
will you support the android tablet version

### DISCUSSION THREAD

**Discussion by Angel0726** [AppFlowy-IO/AppFlowy#486] [General]
will you support the android tablet version？

**Comment by JoelMon** [comment]
This answer is related to your question: https://github.com/AppFlowy-IO/AppFlowy/discussions/540#discussioncomment-2889031

**Reply by Angel0726** [reply]
i saw it. but the Collaborator didn't answer you

**Reply by JoelMon** [reply]
@annieappflowy said that syncing between devices will begin to be developed stating in version v0.0.5. So not yet.

**Comment by annieappflowy** [comment]
@Angel0726 we'll consider it in the long run but it's not included in our short-term [roadmap](https://github.com/AppFlowy-IO/AppFlowy/discussions/565).
I'm converting it into a [FR] issue for better tracking it.
